### DISCUSSION TITLE
How can I edit the texts from the device registration process?

### DISCUSSION THREAD

**Discussion by RedstonePfalz** [zitadel/zitadel#8211] [Q&A] [answered]
Hello,
How can I edit the texts from the device registration process?
I want to edit them because the German texts are not grammatically correct.
And can I change the device name on this screen? It looks weird when the client ID is there instead of the client name.
![image](https://github.com/zitadel/zitadel/assets/55254344/bec49347-4012-4855-bc5b-5390ed432672)

**Comment by hifabienne** [comment, marked as answer]
Hi @RedstonePfalz 
Those texts can be found here: https://github.com/zitadel/zitadel/blob/main/internal/api/ui/login/static/i18n/de.yaml
If you like you can send us directly a pr, so we can apply this for everyone.

As far as I know at the moment the client id is fixed here but @muhlemmer or @livio-a might know more

**Reply by RedstonePfalz** [reply]
Thank you
